### PR TITLE
[nnfwapi] Add while_dynamic model test

### PR DESCRIPTION
**Pull request by wateret**
Add while_dynamic model test to test WHILE op + dynamic tensors.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: tests/nnfw_api/src/ModelTestDynamicTensor.cc [hunk 1]
```diff
@@ -452,3 +452,33 @@ TEST_F(TestDynamicTensorApplyTensorInfoUnaryOp, set_input_tensorinfo_after_compi
...
+const static std::vector<float> output0{ 0.0388205424, 0.0426156297, 0.0980401114, 0.0568757951, 0.1230962500, 0.0412184112, 0.0595490113, 0.4391007423, 0.0377574340, 0.0629260018 };
```
Comment (wateret) [inline tests/nnfw_api/src/ModelTestDynamicTensor.cc:460]: Data is from `keras_rnn` package's metadata/tc dir

File: tests/nnfw_api/src/ModelTestDynamicTensor.cc [hunk 2]
```diff
@@ -452,3 +452,33 @@ TEST_F(TestDynamicTensorApplyTensorInfoUnaryOp, set_input_tensorinfo_after_compi
...
+const static std::vector<float> keras_rnn_output0{ 0.0388205424, 0.0426156297, 0.0980401114, 0.0568757951, 0.1230962500, 0.0412184112, 0.0595490113, 0.4391007423, 0.0377574340, 0.0629260018 };
```
Comment (wateret) [inline tests/nnfw_api/src/ModelTestDynamicTensor.cc:460]: Data is from keras_rnn package's metadata/tc dir

**Comment by ragmani** [comment]
@wateret 
Could you change the test name to `while_dynamic`?

And I have a question.
Would this test run on CI?

**Comment by wateret** [comment]
> @wateret
> Could you change the test name to `while_dynamic`?

OK!

> And I have a question.
> Would this test run on CI?

Yes!

**Pull request merged**